### DISCUSSION TITLE
Batterieüberwachung abschalten

### DISCUSSION THREAD

**Discussion by Cirrus73** [hoylabs/OpenDTU-OnBattery#884] [General]
Ich habe eine Konfiguration mit 2 Balkonkraftwerkspeichern und OpenDTU on Battery laufen.

Die beiden Speicher hängen jeweils an einen Anschluss eines HM600.

Das funktioniert super, bis auf einen Punkt.

Die Speicher haben ihre eigene Steuerung, was gewollt dazu führt, dass die Ausgänge abgeschaltet werden, z.B. wenn die Entladeschwelle erreicht ist.

Hier schießt mir aber die Batterieüberwachung der DTU quer.

Ich muss bei den Einstellungen des Dynamic Power Limiters die Kanal ID angeben.

Das ist der Speicher, auf den die Batterieüberwachung wirkt.

Gebe ich zum Beispiel CH0 an, wird der WR abgeschaltet, wenn der Speicher 1 den Ausgang abschaltet.

Wenn dann der Speicher 2 noch Energie hat, kann er diese nicht mehr entladen.

Wie kann man diese Abschaltung unterdrücken? Diese ist bei mir nicht nötig, da die Speicher das selber regeln.

Vielen Dank für eure Antworten

**Comment by Manos1966** [comment]
Ich habe keine Antwort zu deiner Frage, ich frage mich aber, wie die minus Verbindung im Inneren des HM600 funktioniert...

Sind deine zwei Batterie Speicher gleicher Bauart (Marke, Modell)?

**Comment by spcqike** [comment]
am einfachsten wäre es, wenn du im DPL vorgibst, dass du nur solarpaneele hast. 

Das blöde: dann kannst du (so glaube ich) keinen geregelten Neustart des Wechselrichters mehr erzwingen. 

früher konnte man die Spannungen frei definieren, ich weiß nicht, ob du auch 0V als Grrenzspannung hinterlegen kannst.

> ich frage mich aber, wie die minus Verbindung im Inneren des HM600 funktioniert

zwei Batterien sind 2 unabhängige Spannungsquellen. da gibt es kein Problem, wie es auch bei 2 Solarpaneelen keines gibt.

**Reply by Manos1966** [reply]
@spcqike  🤦‍♂️ Stimmt!

**Comment by schlimmchen** [comment]
> ich weiß nicht, ob du auch 0V als Grrenzspannung hinterlegen kannst.

Nein, aber 16V. Das sollte reichen, denn drunter funktionieren die Hoymiles eh nicht.

**Comment by Cirrus73** [comment]
Danke für eure Antworten.
Geholfen hat der Update auf die neuste Version, dann war die Option "Inverter is powered by solar modules" vorhanden, und alles klappt wie gewünscht
